pkg/kubectl/cmd/create_pvc.go

```diff
@@ -0,0 +1,209 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/api"
+	internalversionstorage "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+var (
+	pvcLong = templates.LongDesc(i18n.T(`
+		Create a persistent volume claim using specified subcommand`))
+
+	pvcExample = templates.Examples(i18n.T(`
+	    # Create a persistent volume claim named "myclaim" with "ReadWriteOnce" access mode and "1Gi" capacity.
+		kubectl create persistentvolumeclaim myclaim --access-mode=ReadWriteOnce --capacity=1Gi
+
+		# Create a persistent volume claim named "myclaim" with "ReadWriteOnce" access mode, "1Gi" capacity and a storage class named "highquality".
+		kubectl create persistentvolumeclaim myclaim --access-mode=ReadWriteOnce --capacity=1Gi --storage-class=hightquality
+
+		# Create a persistent volume claim named "myclaim" with "ReadWriteOnce" access mode, "1Gi" capacity, requiring a persistent volume named "highqualityvolume".
```

Comment:
I'd like to have some note here that this is a specific use case where user wants to use exact PV and this PV must be created by user manually. It will bypass dynamic provisioning and usual binding procedure.

Reply:
add some message for the help output?

Reply:
yes please